src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+@use './variables' as *;
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: $star-size;
+    width: $star-size;
+    margin-right: $star-margin;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} :nth-child(-n + #{$i}) {
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `:` should be followed by a space. Please add a space after the `:` to follow the style guide.